### PR TITLE
fix: Update iTwinUI-css to 0.58.0

### PR DESCRIPTION
**Pull request by gretanausedaite**
- `iui-modal` -> `iui-dialog`
- added `iui-breadcrumbs-item-overrides` and `iui-breadcrumbs-text` to Breadcrumbs
- Changed `iui-radio-tile-description` to `iui-radio-tile-sublabel`
- Added `flexWrap: 'unset'` to Table header to undo css changes until fix is released.

### CODE REVIEW

File: packages/iTwinUI-react/src/core/Table/Table.tsx [hunk 1]
```diff
@@ -708,25 +708,29 @@ export const Table = <
...
-                            <SvgSortDown
-                              className='iui-icon iui-sort'
-                              aria-hidden
-                            />
-                          ) : (
-                            <SvgSortUp
-                              className='iui-icon iui-sort'
-                              aria-hidden
+                      {(column.Filter || column.canSort) && (
+                        <div className='iui-table-header-actions-container'>
```
Comment (bentleyvk) [inline packages/iTwinUI-react/src/core/Table/Table.tsx:712]: Doesn't look nice 
image

Comment (bentleyvk) [reply packages/iTwinUI-react/src/core/Table/Table.tsx:712]: But it will still look bad with regular non-resizable columns

Comment (gretanausedaite) [reply packages/iTwinUI-react/src/core/Table/Table.tsx]: I added inline style to undo flex-wrap for now. We can merge this update and fix table with future PRs.

File: packages/iTwinUI-react/src/core/Breadcrumbs/Breadcrumbs.tsx [hunk 2]
```diff
@@ -63,19 +63,15 @@ export const Breadcrumbs = React.forwardRef(
     const refs = useMergedRefs(overflowRef, ref);
 
     const Separator = () => (
-      <li className='iui-breadcrumbs-separator' aria-hidden>
+      <li className='iui-breadcrumbs-separator' aria-hidden='true'>
```
Comment (bentleyvk) [inline packages/iTwinUI-react/src/core/Breadcrumbs/Breadcrumbs.tsx:66]: Is `'true'` needed?

File: packages/iTwinUI-react/src/core/RadioTiles/RadioTile.test.tsx [hunk 3]
```diff
@@ -10,40 +10,39 @@ import SvgPlaceholder from '@itwin/itwinui-icons-react/cjs/icons/Placeholder';
...
-  expect(container.querySelector('label > .iui-radio-tile')).toBeTruthy();
-  const label = container.querySelector('.iui-label') as HTMLElement;
+  expect(container.querySelector('.iui-radio-tile')).toBeTruthy();
+  const label = container.querySelector('.iui-radio-tile-label') as HTMLElement;
   expect(label).toBeTruthy();
   expect(label.textContent).toBe('My label');
 });
 
-it('should render radio tile with label and description', () => {
+it('should render radio tile with label and sublabel', () => {
```
Comment (bentleyvk) [inline packages/iTwinUI-react/src/core/RadioTiles/RadioTile.test.tsx:27]: Prop is `description` so could just left "description" everywhere

File: packages/iTwinUI-react/src/core/Breadcrumbs/Breadcrumbs.tsx (hunk 2 above)
Comment (gretanausedaite) [reply packages/iTwinUI-react/src/core/Breadcrumbs/Breadcrumbs.tsx]: Nope

File: packages/iTwinUI-react/src/core/RadioTiles/RadioTile.test.tsx (hunk 3 above)
Comment (gretanausedaite) [reply packages/iTwinUI-react/src/core/RadioTiles/RadioTile.test.tsx]: Changed to description

Comment (bentleyvk) [reply packages/iTwinUI-react/src/core/RadioTiles/RadioTile.test.tsx]: I wanted that you revert all your renames, just leave "description" as it was before.

File: packages/iTwinUI-react/src/core/RadioTiles/RadioTile.tsx [hunk 4]
```diff
@@ -31,7 +30,7 @@ export type RadioTileProps = {
 /**
  * RadioTile component to be used in RadioTileGroup component
  * @example
- * <RadioTile label='My tile' description='Some info' icon={<SvgSmileyHappy />} />
+ * <RadioTile label='My tile' sublabel='Some info' icon={<SvgSmileyHappy />} />
```
Comment (bentleyvk) [inline packages/iTwinUI-react/src/core/RadioTiles/RadioTile.tsx:33]: Undo

**Review by bentleyvk — APPROVED** [review]

**Pull request merged**